client/packages/glsp-sprotty/src/features/tool-feedback/model.ts

```diff
@@ -0,0 +1,58 @@
+/*******************************************************************************
+ * Copyright (c) 2018 EclipseSource
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ *  Philip Langer - initial API and implementation
+ ******************************************************************************/
+
+import { Command, CommandExecutionContext, CommandResult, SModelRoot } from "sprotty/lib";
+
+export abstract class FeedbackCommand extends Command {
+
+    abstract execute(context: CommandExecutionContext): SModelRoot;
+
+    undo(context: CommandExecutionContext): CommandResult {
+        return context.root;
+    }
+
+    redo(context: CommandExecutionContext): CommandResult {
+        return context.root;
+    }
+}
+
+export function applyCssClassesToRoot(context: CommandExecutionContext, cssClasses: string[]): SModelRoot {
```

Comment:
Maybe this utility functions should be refactored to the existing smodel utils  file ("src/lib/utils/smodel-util.ts")